### PR TITLE
added sitemap for docsearch

### LINKED ISSUE #210: Search Issues
The search feature seems out of synch with the docs content. Users are also taken to the introduction page, not the actual content page. 

Here are examples:

1. If the user search `kuberetes auth sign` they get taken to the intro page
image

2. No search results with teleport-kube-agent which is in the Multiple Clusters.
3. Search Database RBAC and you don't get the RBAC page when you select RBAC, you get the intro page.

### PR DESCRIPTION
**Pull request by Alqanar**
Fix #210

### CODE REVIEW

File: utils/paths.ts [hunk 1]
```diff
@@ -60,7 +60,27 @@ export const generateSitemap = () => {
 
   sitemapGenerator({
     pages: [...sitePages, ...currentDocPages],
-    path: resolve("public", "static", "sitemap.xml"),
+    path: resolve("public", "sitemap.xml"),
```
Comment (iAdramelk) [inline utils/paths.ts:63]: Cole in #420 already renamed it (we need it named differently from `sitemap.xml` to merge hugo's one and ours), so let's leave this one outside the scope of this PR.

File: utils/paths.ts [hunk 2]
```diff
@@ -60,7 +60,27 @@ export const generateSitemap = () => {
...
+  });
+};
+
+export const generateFullSitemap = () => {
+  const sitePages = getNonDocsPaths().map((loc) => ({ loc }));
```
Comment (iAdramelk) [inline utils/paths.ts:68]: We do not need site pages for the Algolia sitemap, we do not want to index them.

File: utils/paths.ts [hunk 3]
```diff
@@ -60,7 +60,27 @@ export const generateSitemap = () => {
...
+        loc: normalizeDocSlug(slug, version),
+      }))
+    );
+  });
+
+  console.log(docPages);
+
+  sitemapGenerator({
+    pages: [...sitePages, ...docPages],
+    path: resolve("public", "algoliasearchmap.xml"),
```
Comment (iAdramelk) [inline utils/paths.ts:83]: Please discuss with Cole how to name them. His name has `_` yous don't. Its not important how exactly to name them, but we want it to be consistent.

**Review by iAdramelk — CHANGES_REQUESTED** [review]

**Review by klizhentas — APPROVED** [review]

File: utils/paths.ts [hunk 4]
```diff
@@ -64,6 +64,25 @@ export const generateSitemap = () => {
...
+      ...getSlugDataListForVersion(version).map(({ slug, version }) => ({
+        loc: normalizeDocSlug(slug, version),
+      }))
+    );
+  });
+
+  console.log(docPages);
+
+  sitemapGenerator({
+    pages: [...docPages],
```
Comment (iAdramelk) [inline utils/paths.ts:81]: Nit: 

```suggestion
    pages: docPages,
```

**Review by iAdramelk — APPROVED** [review]

**Pull request merged**